pkg/controller/user/create.go

```diff
@@ -81,16 +81,9 @@ func (c *Controller) HandleCreate() http.Handler {
 		}
 
 		// Create firebase user first, if this fails we don't want a db.User entry
-		if created, err := user.CreateFirebaseUser(ctx, c.client); err != nil {
-			flash.Alert("Failed to create user: %v", err)
+		if _, err := c.createFirebaseUser(ctx, user); err != nil {
 			c.renderNew(ctx, w)
```

Comment:
I don't think we should swallow this error.

Reply:
it's not swallowed. c.createFirebaseUser does the flash now (the flash is the same for the places that use the helper)

Reply:
ah i see